README.md

```diff
@@ -77,6 +77,44 @@ const response = await firestoreService.get('regions/32');
 
 The response will have the information about the region with id 32.
 
+#### Tools to make queries
+
+Firestore service allows you to use certain tools to manipulate your data in the query. For this you should add a body into the request.
+
+- Limit
+
+The response will only have 20 "regions"
+
+```js
+const response = await firestoreService.get('regions/32', { limit: 20});
```

Comment:
this differs from what rest apis do, you should never add a body in a get request. to send data using a get request you should use query parameters. the code should look like this:
```
const response = await firestoreService.get('regions/32?limit=20');
```

Reply:
You can check the [apisauce documentation](https://github.com/infinitered/apisauce#calling-the-api) in which you can pass as a second param some values that (in case of the get will be parsed into a param URL. However, I think it must accept both uri params and obeject